### ISSUE TITLE
Bug: Tanker broken on command

### ISSUE THREAD

**Issue by dixonyant** [dcs-liberation/dcs_liberation#197]
When using combined arms to move tanker to new waypoint, the tanker will stop refueling task and will be removed from the radio menu.

Desired behavior:
Tanker should be able to be moved to new waypoint, resuming refueling task, similar to how drones can be moved and they will auto lase targets in the new vicinity.

**Comment by DanAlbert** [comment]
This isn't unique to liberation, is it? I think every DCS mission does that. Refueling is a waypoint action, so if you retask the tanker they stop doing it.

**Comment by dixonyant** [comment]
@DanAlbert Is it possible using scripting to add the task back on maybe a switched condition?

**Comment by DanAlbert** [comment]
Presumably.

**Comment by MaHuJa** [comment]
Yes, it is possible. But reenabling the tanking when it's not needed may have undesirable side effects, like aborting ongoing fueling, if not now then some DCS update later. So this may need to be triggered manually. And now the moving of the tanker becomes a task with multiple disjoint steps. Even if the docs are written people are unlikely to read them, and even if they do the UX is bad.

You also need to reenable the orbit. When it reaches the last waypoint in its new waypoint list, it's effectively given an RTB command.

**Comment by Hornet2041** [comment]
working around combined arms commands breaking tasking is not currently supported but could added in the future, possibly...
